config/betamocks/services_config.yml

```diff
@@ -936,33 +936,3 @@
     - :method: :post
       :path: "/profile-service/profile/v3/*/*"
       :file_path: "vet360/profile-service/default"
-
-# GIDS
-- :name: "GIDS"
```

Comment:
Do you have a plan to remove the mockdata from the vets-api-mockdata repo as well?

Reply:
Yes I removed them in [this PR](https://github.com/department-of-veterans-affairs/vets-api-mockdata/pull/570/files)